hack/release.sh

```diff
@@ -32,11 +32,20 @@ COMPONENTS=(
 readonly COMPONENTS
 
 function build_release() {
+   # Update release labels if this is a tagged release
+  if [[ -n "${TAG}" ]]; then
```

Comment:
who sets this TAG env variable?

Reply:
It's set by the release helpers when both `--tag-release` and `--version` are set.